gazebo/create_dockerfiles.py

```diff
@@ -16,6 +19,35 @@
 from ros_buildfarm.docker_common import OrderedLoad
 
 
+def checkVersion(data):
+    """Checks current package_version from online package index"""
+
+    # Determine URL
+    urlTemplate = string.Template(
+        """
+        http://packages.osrfoundation.org/gazebo/$os_name/dists/$os_code_name/main/binary-$arch/Packages
```

Comment:
What vocab would you use to call `stable/prerelease`? 
I'll make an alias for it so it could be `$os_name-$sothing`